File: ruby/linear_search.rb
def linear_search(arr, n, key)
  for i in 0...n
    if key == arr[i]
      return i
    end
  end
  return -1
end

array = [5,2,4,6,1,3,0]
size = array.length
v = 0
result = linear_search(array, size, v)

if result == -1
  puts "No Match found"
else
  puts "#{v} found at position #{result+1}"
end

File: js/linear_search.js
function linear_search(arr, n, key){
    for(i=0;i<n;i++){
        if(arr[i]==key){
            return i
        }
    }
    return -1
}

const array = [31, 41, 26, 41, 58, 59]
const size = array.length
const v = 26
const result = linear_search(array, size, v)

if(result == -1){
    console.log('No match Found')
}else{
    console.log(`${v} is in position ${result+1}`)
}
